logic/src/commonMain/kotlin/com/wire/kalium/logic/data/message/MessageMapper.kt

```diff
@@ -371,10 +371,15 @@ private fun MessagePreviewEntityContent.toMessageContent(): MessagePreviewConten
     is MessagePreviewEntityContent.ConversationNameChange -> MessagePreviewContent.WithUser.ConversationNameChange(adminName)
     is MessagePreviewEntityContent.Knock -> MessagePreviewContent.WithUser.Knock(senderName)
     is MessagePreviewEntityContent.MemberChange -> when (type) {
-        MessageEntity.MemberChangeType.ADDED -> MessagePreviewContent.WithUser.MembersAdded(adminName = adminName, count = count)
-        MessageEntity.MemberChangeType.REMOVED -> MessagePreviewContent.WithUser.MembersRemoved(adminName = adminName, count = count)
+        MessageEntity.MemberChangeType.ADDED -> MessagePreviewContent.WithUser.MembersAdded(
+            adminName = adminName,
```

Comment:
Is `adminName` the name of the person who added/removed someone? Maybe we could use better name here as conversation can have multiple admins, also if someone who is not an admin leaves, this system message type is also created even if he's not an admin 😉

Reply:
Renamed to sender